dev/tasks/python-wheels/travis.osx.yml

```diff
@@ -74,8 +72,8 @@ install:
 
   # crossbow dependencies for deployment
   - unset MACOSX_DEPLOYMENT_TARGET
-  - pip3 install click ruamel.yaml setuptools_scm github3.py toolz requests[security]
-  - python3 arrow/dev/tasks/crossbow.py
+  - /usr/local/bin/pip3 install click ruamel.yaml setuptools_scm github3.py toolz requests[security]
+  - /usr/local/bin/python3 arrow/dev/tasks/crossbow.py
```

Comment:
There were ssl issues in the 3.5 build. I hope using the brew python will solve that.